entrypoints/injected/components/Code.tsx

```diff
@@ -149,7 +149,7 @@ export const CodeArea = memo((props: { minimized?: boolean }) => {
           onClick={(e) => e.stopPropagation()}
         >
           {unoResult?.map((u, index) => (
-            <>
+            <Fragment key={u.title}>
```

Comment:
What are the benefits of using `Fragment`?

Reply:
Because it will throw a warning for a unique "key"
![image](https://github.com/zouhangwithsweet/fubukicss-tool/assets/32429378/84546368-6a4e-4938-865c-1359440ee5e4)